### PR TITLE
pulledpork 0.7.2

### PR DESCRIPTION
**Pull request by nmani**
Updated to use the most forked/active github repository for pulledpork. This repo includes numerous updates and commits over the old svn repo.

https://github.com/shirkdog/pulledpork/

### CODE REVIEW

**Comment by DomT4** [comment]
> Updated to use the most forked/active github repository for pulledpork. This repo includes numerous updates and commits over the old svn repo.

If upstream hasn't "blessed" the new fork we don't generally allow it to take over the formula, because it violates the principle of least surprise by updating the user's existing formula to an entirely different source and maintainer. We'll need something from upstream here.

**Comment by nmani** [comment]
While I see your point, the last commit by the upstream developer was 9/11/2013 (source: https://code.google.com/archive/p/pulledpork/source/default/commits?page=1) . It seems like the upstream developer has abandoned this project and his repo is the de facto successor.

**Comment by tdsmith** [comment]
We could call this `pulledpork-shirkdog` or something similar, or we can pull this as-is if you can get a statement from JJ Cummings (cc homebrew-discuss@googlegroups.com) designating this as the successor project.

File: Library/Formula/pulledpork.rb [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
   desc "Snort rule management"
-  homepage "https://code.google.com/p/pulledpork/"
-  url "https://pulledpork.googlecode.com/files/pulledpork-0.7.0.tar.gz"
-  sha256 "f60c005043850bb65a72582b9d6d68a7e7d51107f30f2b3fc67e607c995aa1a8"
-  head "http://pulledpork.googlecode.com/svn/trunk/"
+  homepage "https://github.com/shirkdog/pulledpork"
+  url "https://github.com/shirkdog/pulledpork/archive/0.7.2.tar.gz"
+  sha256 "f4b72d03d1ae509049d771fe646c421ec1a877a4a07addc5f6046a3d614e4be7"
+
+  head "https://github.com/shirkdog/pulledpork.git", :branch => "master"
```
Comment (ilovezfs) [inline Library/Formula/pulledpork.rb]: master appears to already be the default branch

Comment (mikemcquaid) [inline Library/Formula/pulledpork.rb]: (so you can remove `:branch => "master"`)

**Comment by mikemcquaid** [comment]
> We could call this pulledpork-shirkdog or something similar, or we can pull this as-is if you can get a statement from JJ Cummings (cc homebrew-discuss@googlegroups.com) designating this as the successor project.

:+1:

**Comment by DomT4** [comment]
Closing here, but we'll take a look at a new formula submission for this if you wish to submit one. Thank you for the submission to Homebrew :heart:

**Pull request closed without merging**